functions/date-time-offset/1.0/index.js

```diff
@@ -0,0 +1,83 @@
+import addSeconds from 'date-fns/addSeconds';
+import addHours from 'date-fns/addHours';
+import addMinutes from 'date-fns/addMinutes';
+import addDays from 'date-fns/addDays';
+import addWeeks from 'date-fns/addWeeks';
+import addMonths from 'date-fns/addMonths';
+import addYears from 'date-fns/addYears';
```

Comment:
Why not import it like `import { addSeconds,  addHours } from date-fns` I think bundle size is not really an issue here. Especially since you already using this on line 8